src/projectile.py

```diff
@@ -37,26 +38,29 @@ def __init__(
         self,
         sprite: LinkedSprite[Projectile],
         sprite_lists: SpriteLists,
-        speed: float,
-        radians: float,
         damage: float,
     ):
         self.sprite = sprite
         sprite.owner = self
         self.sprite_lists = sprite_lists
-        self.speed = speed
-        self.radians = radians
         self.damage = damage
-        self.explodes = False
         self.exists = False
-        self.sprite_rotation_offset = 0
-        self.setup()
+        self.speed = 0
+        self.angle_of_motion = 0
 
-    def setup(self):
+    def setup(self, muzzle_location : Tuple[float, float, float], speed: float,
+        angle_of_motion: float, sprite_rotation_offet:float = 0, explodes: bool = False):
```

Comment:
This line makes me suspect the formatter should be re-run?  Still waiting on Mike S's forthcoming CI work to verify this stuff automatically for us.

Reply:
Fixed. A ctrl-S took care of it even though I have auto save on